.github/workflows/all_plugins.yaml

```diff
@@ -44,8 +44,10 @@ jobs:
         uses: subosito/flutter-action@refs/pull/79/head
       - name: 'Install Tools'
         run: |
+          sudo apt update
+          sudo apt install -y clang-format
           flutter pub global activate melos
-          flutter pub global activate flutter_plugin_tools
+          pub global activate flutter_plugin_tools
```

Comment:
I highly recommend you pin a version here (see [here](https://github.com/flutter/packages/blob/master/.cirrus.yml#L9-L11) for instance). There isn't any kind of guarantee going forward that there won't be breaking changes to the tool's API.

Reply:
Good point, Done.